smart/config.py

```diff
@@ -332,13 +334,12 @@ class SolverConfig(BaseConfig):
     :param final_t: End time of simulation
     :param use_snes: Use PETScSNES solver if true, else use DOLFINs NewtonSolver
     :param snes_preassemble_linear_system: If True separate linear components during assembly
+        (in development, untested)
     :param initial_dt: Initial time-stepping
     :param adjust_dt: A tuple (t, dt) of floats indicating when to next adjust the
         time-stepping and to what value
     :param dt: Number of digits for rounding `dt`
     :param print_assembly: Print information during assembly process
-    :param dt_decrease_factor:
-    :param dt_increase_factor:
     :param attempt_timestep_restart_on_divergence: Restart snes solver if it diverges
```

Comment:
We have used [Google style](https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html) for the rest of the docstrings. Would be good to be consistent about the style (but perhaps not the main priority)